linode/manager
# fix: [M3-7245] - Replaced 'Devices' with 'Services' in Firewall Page

## Description 📝
For the Firewalls, we decided to use the term "Services" instead of "Devices" since CF will eventually expand to LKE, Object Storage, etc.

## Changes  🔄
- "Devices" --> "Services" in 
   - Firewall Landing table 
   - Create Firewall drawer
   - Add Linode to Firewall
   - Add NodeBalancer to Firewall 
- Updated routing so that '/add' opens up the Add NodeBalancer drawer and the Add Linode Drawer

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-10-10 at 12 27 01 PM](https://github.com/linode/manager/assets/139489745/4e460128-2ff5-47ec-90b0-f9fde61970bd) | ![Screenshot 2023-10-10 at 11 40 27 AM](https://github.com/linode/manager/assets/139489745/baf41f86-dd1c-4d3b-acdc-0d0bd30796e0) |
| ![Screenshot 2023-10-10 at 12 27 16 PM](https://github.com/linode/manager/assets/139489745/333b52a7-74a7-41b3-8746-42593489f37e) | ![Screenshot 2023-10-10 at 11 40 47 AM](https://github.com/linode/manager/assets/139489745/6d32c4d2-338f-4745-a3ee-645f7a26d420) |
| ![Screenshot 2023-10-10 at 12 27 43 PM](https://github.com/linode/manager/assets/139489745/410c5411-5075-4649-8935-fd6439a400d4) | ![Screenshot 2023-10-10 at 11 41 13 AM](https://github.com/linode/manager/assets/139489745/56273d8b-49e5-4438-91e3-6bb21df6a11d) |
| ![Screenshot 2023-10-10 at 12 27 57 PM](https://github.com/linode/manager/assets/139489745/e94f8c69-9429-401f-8a1b-ccac17be048d) | ![Screenshot 2023-10-10 at 11 41 26 AM](https://github.com/linode/manager/assets/139489745/60f01bea-1527-4b47-8423-f604f37f3bb3) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- There are no Prerequisites, you can use either your dev or prod account.

### Reproduction steps
- Checkout the feature/firewall-nodebalancer branch
Firewall Landing Table:
- Navigate to http://localhost:3000/firewalls
- Ensure the 4th column is named "Devices"
Firewall Create Drawer:
- Navigate to http://localhost:3000/firewalls/create
- Ensure the name of the second section is 'Assign devices to the Firewall' and 'devices' is also referred to below the section header
Add Linodes to Firewall Drawer:
- Navigate to http://localhost:3000/firewalls/[firewall_id]/linodes
- Click `Add Linode to Firewall`
- Ensure the word 'Devices' is used in the drawer
Add Linodes to Firewall Drawer:
- Navigate to http://localhost:3000/firewalls/[firewall_id]/nodebalancers
- Click `Add NodeBalancer to Firewall`
- Ensure the word 'Devices' is used in the drawer

### Verification steps 
(How to verify changes)
- Checkout this PR branch
- Ensure "Devices" have been swapped with "Services" in 
   - [ ] Firewall Landing table 
   - [ ] Create Firewall drawer
   - [ ] Add Linode to Firewall
   - [ ] Add NodeBalancer to Firewall 
- [ ] Ensure that http://localhost:3000/firewalls/[firewall_id]/linodes/add opens up the `Add Linode Drawer`
- [ ] Navigate to http://localhost:3000/firewalls/[firewall_id]/linodes and ensure that clicking on `Add Linodes To Firewall` opens the drawer and updates the url by adding '/add'
- [ ] Ensure that http://localhost:3000/firewalls/[firewall_id]/nodebalancers/add opens up the `Add NodeBalancer Drawer`
- [ ] Navigate to http://localhost:3000/firewalls/[firewall_id]/nodebalancers and ensure that clicking on `Add Linodes To Firewall` opens the drawer and updates the url by adding '/add'

## As an Author I have considered 🤔
